src/RpcServer/RpcServer.Wallet.cs

```diff
@@ -138,11 +138,14 @@ private JObject OpenWallet(JArray _params)
             return true;
         }
 
-        private void ProcessInvokeWithWallet(JObject result)
+        private void ProcessInvokeWithWallet(JObject result, Cosigners cosigners = null)
         {
-            if (wallet != null)
+            if (wallet != null && cosigners != null)
```

Comment:
I think that we should throw an exception if doesn't have the wallet or cosigners, otherwise it doesn't do anything

Reply:
Added :-)

Reply:
If wallet or cosigners are null, nothig happend

Reply:
If wallet or cosigners are null, it just return the stack, don't include the tx, need anything else?

Reply:
But the rpc client will notice that this happend?

Reply:
Set the default tx to null ?